src/ocaml/typing/typecore.ml

```diff
@@ -5408,7 +5408,7 @@ and type_label_exp create env loc ty_expected
   if is_poly then check_univars env "field value" arg label.lbl_arg vars;
   (lid, label, {arg with exp_type = instance arg.exp_type})
 
-and type_argument ?explanation ?recarg env sarg ty_expected' ty_expected =
+and type_argument_ ?explanation ?recarg env sarg ty_expected' ty_expected =
```

Comment:
what is the difference between `type_argument` and `type_argument_` ? could we find more meaningful names? 
And it looks strange that we touch to something inside `src/ocaml/typing`? Is this related to type recovery?

Reply:
Oh ok! In fact `type_argument_` is the OCaml one and `type_argument` is a wrapper dealing with Merlin specific stuff? Make sense.